### PR TITLE
story/EUI-7032

### PR DESCRIPTION
**Pull request by Kartheek-S**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-7032

### Change description ###
Work Allocation R3.5 - Roles Access tab- Add CTSC section and role assignment journey

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x ] No
```

### CODE REVIEW

**Review by DavidJayakumar — APPROVED** [review]
LGTM

File: src/work-allocation/containers/task-list-wrapper/task-list-wrapper.component.ts [hunk 1]
```diff
@@ -5,6 +5,8 @@ import { FeatureToggleService, FilterService, FilterSetting } from '@hmcts/rpx-x
 import { Store } from '@ngrx/store';
 import { Observable, of, Subscription } from 'rxjs';
 import { debounceTime, filter, mergeMap, switchMap } from 'rxjs/operators';
+import { AppConstants } from 'src/app/app.constants';
```
Comment (connorpgpmcelroy) [inline src/work-allocation/containers/task-list-wrapper/task-list-wrapper.component.ts:8]: Need to remove src

**Review by connorpgpmcelroy — CHANGES_REQUESTED** [review]
Mostly looks good, thanks for the wa-service-config change on roles-and-access component, but need to remove src from one of the imports please

**Review by connorpgpmcelroy — APPROVED** [review]
LGTM

**Pull request merged**